src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -46,6 +51,21 @@ def load_arguments(self, _):
         c.argument('kube_context', options_list=['--kube-context'], help='Kubconfig context from current machine.')
         c.argument('arc_agent_version', options_list=['--agent-version'], help='Version of agent to update the helm charts to.')
 
+    with self.argument_context('connectedk8s enable-features') as c:
+        c.argument('cluster_name', options_list=['--name', '-n'], id_part='name', help='The name of the connected cluster.')
+        c.argument('kube_config', options_list=['--kube-config'], help='Path to the kube config file.')
+        c.argument('kube_context', options_list=['--kube-context'], help='Kubconfig context from current machine.')
```

Comment:
I had observed that cli does not work with kube-conext parameter. Let's test cli with both kube-config and kube-context parameter if not already.

Reply:
When did you observe this, i remember there was an issue long back in private preview of cluster connect, which was resolved at that time in 0.3.5.

Still, will verify again in this also.

Reply:
In which command?

Reply:
Have you observed this in private preview version 0.3.5 , or the public cli?

I remember there was an issue long back in cluster connect private preview phase with kube-context and the aad-profile related, which was resolved in 0.3.5.

Still, we will validate in this.

Reply:
Anumita validated that it is working with enable_features and disable_features too.